internal/framework/schemas.go

```diff
@@ -97,6 +101,28 @@ func IsImmutableAttribute(oas *openapi3.Schema) bool {
 	return IsExtensionSet(oas, "x-immutable")
 }
 
+func GetStringValidators(oas *openapi3.Schema) []validator.String {
+	var validators []validator.String
+
+	format := oas.Format
+	if format == "date-time" {
+		timestampPattern := "^([0-9]+)-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])[Tt]([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9]|60)(([Zz])|([\\+|\\-]([01][0-9]|2[0-3]):[0-5][0-9]))$"
```

Comment:
whoever wrote this regex is smarter than me - but just curious, why would be "60" a valid minute value? Is this a rounding thing or a bug workaround from the past? No need to make code changes here.

Reply:
Source: https://gist.github.com/marcelotmelo/b67f58a08bee6c2468f8

60 is not a valid minute value, but may be a valid leap second.